build.gradle

```diff
@@ -146,6 +151,19 @@ jar {
     from sourceSets.generated.output
 }
 
+liquibase {
+    activities {
+        main {
+            changeLogFile 'src/main/resources/db/changelog.xml'
+            url 'jdbc:postgresql://127.0.0.1:5432/datarepo'
```

Comment:
Can the JDBC path be pulled from the environment? This would require that all environments have the same port and database name.

The configuration on my Mac is running both the local postgres at port 5454 and the cloud sql proxy of my test environment at port 5432.

My solution in the junit code was to load properties from a path; one of those properties is the JDBC url. An alternative would be to change the unit test code and this code to use something from the environment.